### PR TITLE
[15.0][MIG] account_bank_statement_import_move_line: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa  TT41766

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza please review

File: account_bank_statement_import_move_line/models/account_move_line.py [hunk 1]
```diff
@@ -19,11 +19,16 @@ def _prepare_statement_line_vals(self, statement):
             "amount": amount,
             "partner_id": self.partner_id.id,
             "statement_id": statement.id,
-            "ref": self.ref,
+            "payment_ref": self.ref,
             "date": self.date_maturity,
-            "amount_currency": self.amount_currency,
             "currency_id": self.currency_id.id,
         }
+        if statement.company_id.currency_id != self.company_id.currency_id:
```
Comment (pedrobaeza) [inline account_bank_statement_import_move_line/models/account_move_line.py]: It should be journal currency, not company currency.

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza please review

File: account_bank_statement_import_move_line/models/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        vals = {
+            "name": self.name or "?",
+            "amount": amount,
+            "partner_id": self.partner_id.id,
+            "statement_id": statement.id,
+            "payment_ref": self.ref,
+            "date": self.date_maturity,
+            "currency_id": self.currency_id.id,
+        }
+        if statement.journal_id.currency_id != self.journal_id.currency_id:
```
Comment (pedrobaeza) [inline account_bank_statement_import_move_line/models/account_move_line.py]:
```suggestion
        if statement.currency_id != self.currency_id:
```

Comment (ernesto-garcia-tecnativa) [reply account_bank_statement_import_move_line/models/account_move_line.py]: Resolved

**Review by pedrobaeza — APPROVED** [review]

**Comment by ernesto-garcia-tecnativa** [comment]
@victoralmau please review

File: account_bank_statement_import_move_line/wizards/account_statement_line_create.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                    }
+                )
+        return res
+
+    def _prepare_move_line_domain(self):
+        self.ensure_one()
+        domain = [
+            ("reconciled", "=", False),
+            ("account_id.internal_type", "in", ("payable", "receivable")),
+            ("company_id", "=", self.env.user.company_id.id),
```
Comment (victoralmau) [inline account_bank_statement_import_move_line/wizards/account_statement_line_create.py]:
```suggestion
            ("company_id", "=", self.env.company.id),
```

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**